apps/daimo-mobile/src/view/screen/InviteScreen.tsx

```diff
@@ -267,6 +279,14 @@ function ReferralButtonsFooter({
     } else return "";
   })();
 
+  const shareFarcaster = () => {
+    console.log(`[INVITE] share on farcaster`);
+    const frameUrl = `https://daimo.com/frame/invite/${account.address}`;
+    const message = encodeURIComponent(frameUrl);
+    Linking.openURL(`https://warpcast.com/~/compose?text=${message}`);
```

Comment:
do you need `embeds[]=` here? https://docs.farcaster.xyz/reference/warpcast/cast-composer-intents